gammapy/image/tests/test_lists.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from gammapy.datasets import FermiGalacticCenter
+import numpy as np
+
+
+def test_to_cube():
+    sky_cube_original = FermiGalacticCenter.diffuse_model()
+    image_list = sky_cube_original.to_image_list()
+    assert np.array_equal(image_list.skymaps[0].data, sky_cube_original.data[0])
```

Comment:
Comparing arrays with == gives 
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all(), thanks for pointing me assert_array_equal!